Source/WebCore/layout/layouttree/LayoutBox.cpp

```diff
@@ -169,7 +169,7 @@ bool Box::establishesFlexFormattingContext() const
 
 bool Box::establishesGridFormattingContext() const
 {
-    return isGridBox();
+    return isGridFormattingContext();
```

Comment:
IMO this function definition should just have the code from the body of `isGridFormattingContxt()`